srcs/client/src/Components/Tools/Box.css

```diff
@@ -127,23 +231,59 @@
 	border-radius: 3vw;
 	margin: 1vw;
 	padding: 0.9vw;
-	max-height: 100%;
+	max-height:40vh;
+	min-height : 20vh;
 }
 
+table.blueBoxFriend
```

Comment:
I think classes with a specific indication to a component (friend, ladder, profil, ...) should be close to the components they are styling and that we should keep common locations like this one (tools/box.css) for generic classes that we can share on multiple components.